googletest/include/gtest/gtest.h

```diff
@@ -601,6 +601,9 @@ class GTEST_API_ TestResult {
   // Returns the elapsed time, in milliseconds.
   TimeInMillis elapsed_time() const { return elapsed_time_; }
 
+  // Return the number of assert executed with no errors in this test.
+  size_t success_assert_count() const { return success_assert_count_; }
```

Comment:
even though sizet_t is correct-er we should be using int to prevent bugs. For example you have in gtest.cc this line " test_info.result()->total_part_count() + test_info.result()->success_assert_count());"  one is an int the other one is size_t, which will produce undesired behavior.  Please change everything to be int, to be consistent with the existing code and prevent these issues

Reply:
This comment pertains to may places in this PR so please change to int where applicable

Reply:
I am not sure I understand how this can become a problem?
size_t can hold a larger positive integer than int, and even though total_part_count() returns 
an int, it should never become negative? 

I have no problem converting success_assert_count to int but thought size_t a better choice as the number of asserts for a test can easily become huge quickly,
possible wrapping to negative for an int.

Reply:
It is better to be consistent with the existing code

Reply:
ok - will change back to int then